dev/travis/save-logs.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import sys
+import subprocess
+from datetime import datetime, timedelta
+
+def main(file, cmd):
+    print cmd, "writing to", file
+    out = open(file, "w")
```

Comment:
I think the flag should be "a" instead of "w".
Script section does not break after error exit. That'll overwrite error message in "install.txt" with next step's output.
Check http://docs.travis-ci.com/user/customizing-the-build/#Breaking-the-Build